src/Routes/Router.tsx

```diff
@@ -26,17 +26,17 @@ const Router = () => {
   const { userToken } = useSelector((state: RootState) => state.auth);
   const { decodedToken } = useJwt<DecodedToken>(userToken);
 
-  const isAdmin = decodedToken?.role.toLowerCase() === 'admin';
-  const isVendor = decodedToken?.role.toLowerCase() === 'vendor';
-  const isBuyer = decodedToken?.role.toLowerCase() === 'buyer';
+  const isAdmin = decodedToken?.userType.toLowerCase() === 'admin';
```

Comment:
@UwicyezaG  You must use the value from 'role' instead of 'userType', as the token does not contain userType.